Remotion/ObjectBinding/Web/UI/Controls/BocTreeView.cs

```diff
@@ -644,7 +644,7 @@ private void LoadNodesControlStateRecursive (Pair[] nodesState, WebTreeNodeColle
         bool isSelected = (bool)values[3];
         string menuID = (string)values[4];
         WebString text = (WebString)values[5];
-        WebString toolTip = (WebString)values[6];
+        PlainTextString toolTip = PlainTextString.CreateFromText((string)values[6]);
```

Comment:
Why can't we stick the PlainTextString into the values, same as WebString?

And yes, I'm aware that there's boxing involved, but it better preserves the semantics by not going lowlevel at this point.